dashboard/test/ui/features/step_definitions/pd.rb

```diff
@@ -113,6 +163,14 @@ def create_facilitator(course)
   facilitator
 end
 
+def create_applications_in_states(application_count, state_hash)
+  include FactoryGirl::Syntax::Methods
+  application_count.times do
+    teacher = create :teacher, email: "teacher_application_teacher_#{SecureRandom.hex}@code.org"
+    create :pd_teacher1819_application, user: teacher
+  end
+end
```

Comment:
What is this? It appears to be unused, and the `state_hash` is also unused internally.